Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -992,13 +992,26 @@ public override bool CanConvert(Type objectType)
         /// <summary>
         /// Distributed Tracing Options. <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/>
         /// </summary>
+        /// <remarks> Applicable only when Operation level distributed tracing is not disabled i.e. <see cref="Microsoft.Azure.Cosmos.CosmosClientOptions.IsDistributedTracingEnabled"/></remarks>
         internal DistributedTracingOptions DistributedTracingOptions { get; set; }
 
         /// <summary>
-        /// Gets or sets value indicating whether distributed tracing activities (<see cref="System.Diagnostics.Activity"/>) are going to be created for the SDK methods calls and HTTP calls.
-        /// By default true for Preview package
+        /// By default, SDK (Source Name is "Azure.Cosmos.Operation") generates operation level <see cref="System.Diagnostics.Activity"/> for methods calls.
```

Comment:
Other properties that enable/disable use the "Gets or sets the flag...", what is the default is already explained in the `<value>` tag.

```suggestion
        /// Gets or sets the flag to generate operation level <see cref="System.Diagnostics.Activity"/> for methods calls using the Source Name "Azure.Cosmos.Operation".
```